### PR TITLE
Update admonition doc to use screenshot from site

### PR DESCRIPTION
**Pull request by SaschaMann**
Probably could do with a proper alt text but I'm not sure what to put in there.

### CODE REVIEW

**Review by ErikSchierboom — APPROVED** [review]

File: building/markdown/markdown.md [hunk 1]
```diff
@@ -94,7 +94,11 @@ A quick way to distinguish between the two cases when it's unclear - if this is
 We support special types of blocks that can be added to documents to pull out commentary that doesn't fit with the main body of the text.
 They are similar to these examples, seen in the Julia docs:
 
-<img width="500" alt="Screenshot 2021-03-13 at 17 15 04" src="https://user-images.githubusercontent.com/286476/111038207-aca0bd00-841f-11eb-95fb-20a93943d3dd.png">
+<img width="500" alt="Screenshot 2021-10-30 at 13 27 00" src="https://user-images.githubusercontent.com/20866761/139531109-0c0f3ea5-b589-44ff-93ab-d901a45edc56.png">
```
Comment (ErikSchierboom) [inline building/markdown/markdown.md]: Maybe something like:

```suggestion
<img width="500" alt="Markdown note block" src="https://user-images.githubusercontent.com/20866761/139531109-0c0f3ea5-b589-44ff-93ab-d901a45edc56.png">
```

File: building/markdown/markdown.md [hunk 2]
```diff
@@ -94,7 +94,11 @@ A quick way to distinguish between the two cases when it's unclear - if this is
 We support special types of blocks that can be added to documents to pull out commentary that doesn't fit with the main body of the text.
 They are similar to these examples, seen in the Julia docs:
 
-<img width="500" alt="Screenshot 2021-03-13 at 17 15 04" src="https://user-images.githubusercontent.com/286476/111038207-aca0bd00-841f-11eb-95fb-20a93943d3dd.png">
+<img width="500" alt="Markdown note block" src="https://user-images.githubusercontent.com/20866761/139531109-0c0f3ea5-b589-44ff-93ab-d901a45edc56.png">
+
+<img width="500" alt="Screenshot 2021-10-30 at 13 27 00" src="https://user-images.githubusercontent.com/20866761/139531234-9b58b9ab-524b-44fd-bbcb-4e10f2927d27.png">
```
Comment (SaschaMann) [inline building/markdown/markdown.md]:
```suggestion
<img width="500" alt="Markdown caution block" src="https://user-images.githubusercontent.com/20866761/139531234-9b58b9ab-524b-44fd-bbcb-4e10f2927d27.png">
```

File: building/markdown/markdown.md [hunk 3]
```diff
@@ -94,7 +94,11 @@ A quick way to distinguish between the two cases when it's unclear - if this is
...
+
+<img width="500" alt="Screenshot 2021-10-30 at 13 27 00" src="https://user-images.githubusercontent.com/20866761/139531236-72259046-68a9-4934-9440-a730befde155.png">
```
Comment (SaschaMann) [inline building/markdown/markdown.md]:
```suggestion
<img width="500" alt="Markdown advanced block" src="https://user-images.githubusercontent.com/20866761/139531236-72259046-68a9-4934-9440-a730befde155.png">
```

**Review by ErikSchierboom — APPROVED** [review]
Thanks!

**Pull request merged**